e2e-tests/playwright/e2e/plugins/adoption-Insights/adoption-Insights.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { test, expect } from '@playwright/test';
 import { Common } from "../../../utils/common";
 import { UIhelper } from "../../../utils/ui-helper"; 
-import { TestHelper } from "../../../support/pages/adoption-Insights"; 
+import { TestHelper } from "../../../support/pages/adoption-insights"; 
 
 // TODO: replace skip with serial https://issues.redhat.com/browse/RHDHBUGS-1879
 test.describe.skip("Test Adoption Insights", () => {
```

Comment:
You have to re-enable the test to verify it in the CI
```suggestion
test.describe.serial("Test Adoption Insights", () => {
```